tmt/export.py

```diff
@@ -213,7 +222,10 @@ def create_nitrate_case(test):
         category = 'Sanity'
 
     # Create the new test case
-    summary = test.node.get('extra-summary', test.summary)
+    if not os.path.basename(test.fmf_id['url']):
```

Comment:
Let's prevent calling `test.fmf_id()` twice as it does three shell commands under the hood.

Reply:
fixed